fem/lininteg.hpp

```diff
@@ -470,6 +470,25 @@ class VectorFEBoundaryFluxLFIntegrator : public LinearFormIntegrator
                                Vector &b);
 };
 
+/** Class for boundary integration of (f.n, v.n) for vector coefficient f and
+    RT vector test function v. This integrator works with RT spaces defined
+    using the RT_FECollection class. */
+class VectorFEBoundaryNormalLFIntegrator : public LinearFormIntegrator
+{
+private:
+   VectorCoefficient &F;
+   Vector shape;
+
+public:
+   VectorFEBoundaryNormalLFIntegrator(VectorCoefficient &f) : F(f) { }
+
+   virtual void AssembleRHSElementVect(const FiniteElement &el,
+                                       ElementTransformation &Tr,
+                                       Vector &elvect);
```

Comment:
```suggestion
   virtual void AssembleRHSElementVect(const FiniteElement &el,
                                       ElementTransformation &Tr,
                                       Vector &elvect) override;
```
Might also consider removing the `virtual` unless you intend for this class to be derived.

Reply:
Sure, I very much support using `override`s in code. It just does not appear almost anywhere 😬 , but we may start here :wink: .
About `virtual`, I am not that sure. It also appears everywhere and most of the time without a real reason, but it might be seen an element of future extensibility, i.e., the code will not need to be changed when someone adds a derived class 🤔 .

Reply:
If the class is written in such a way that this function is encapsulated, i.e., a derived class overriding this function won't cause issues (e.g., this function writes a value to a member variable that is read by another member function), then I suppose there's no harm in making it `virtual` (aside from the additional pointer and lookup, which is likely negligible or perhaps optimized out).  Otherwise, I consider the need to modify this file as a good thing in that it act as a reminder that this class was not necessarily written such that the function is encapsulated.